Integrations/FortiSIEM/FortiSIEM.py

```diff
@@ -0,0 +1,690 @@
+    report.setAttribute("id", "")
+    report.setAttribute("group", "report")
+    reports.appendChild(report)
+    name = doc.createElement("Name")
+    report.appendChild(name)
+    doc.createTextNode("All Incidents")
+    custScope = doc.createElement("CustomerScope")
+    custScope.setAttribute("groupByEachCustomer", "true")
+    report.appendChild(custScope)
+    include = doc.createElement("Include")
+    if include_value == "all":
+        include.setAttribute("all", "true")
+        custScope.appendChild(include)
+    else:
+        custScope.appendChild(include)
+        include_text = doc.createTextNode(include_value)
+        include.appendChild(include_text)
+    exclude = doc.createElement("Exclude")
+    custScope.appendChild(exclude)
+    description = doc.createElement("description")
+    report.appendChild(description)
+    select = doc.createElement("SelectClause")
+    select.setAttribute("numEntries", limit)
+    report.appendChild(select)
+    attrList = doc.createElement("AttrList")
+    if attr_list:
+        attr_text = doc.createTextNode(str(attr_list))
+        attrList.appendChild(attr_text)
+    select.appendChild(attrList)
+    reportInterval = doc.createElement("ReportInterval")
+    report.appendChild(reportInterval)
+    window = doc.createElement("Window")
+    window.setAttribute("unit", interval_type)
+    window.setAttribute("val", interval)
+    reportInterval.appendChild(window)
+    pattern = doc.createElement("PatternClause")
+    pattern.setAttribute("window", "3600")
+    report.appendChild(pattern)
+    subPattern = doc.createElement("SubPattern")
+    subPattern.setAttribute("displayName", "Events")
+    subPattern.setAttribute("name", "Events")
+    pattern.appendChild(subPattern)
+    single = doc.createElement("SingleEvtConstr")
+    subPattern.appendChild(single)
+    single_text = doc.createTextNode(single_evt_value)
+    single.appendChild(single_text)
+    _filter = doc.createElement("RelevantFilterAttr")
+    report.appendChild(_filter)
+    return doc.toxml()
+
+
+def dumpXML(xmlList, phCustId):
+    param = []
+    for xml in xmlList:
+        doc = parseString(xml.encode('utf-8'))
+        for node in doc.getElementsByTagName("events"):
+            for node1 in node.getElementsByTagName("event"):
+                mapping = {}
+                for node2 in node1.getElementsByTagName("attributes"):
+                    for node3 in node2.getElementsByTagName("attribute"):
+                        item_name = node3.getAttribute("name")
+                        for node4 in node3.childNodes:
+                            if node4.nodeType == Node.TEXT_NODE:
+                                mapping[item_name] = node4.data
+                    if phCustId == "all" or mapping['phCustId'] == phCustId:
+                        param.append(mapping)
+    return param
+
+
+def buildQueryString(args):
+    res_list = []
+    for key in args:
+        if key.find("IpAddr") == -1:
```

Comment:
seems more python this way
```suggestion
        if IpAddr not in key:
```